src/types.d.ts

```diff
@@ -43,6 +43,16 @@ export interface EnterpriseLearnerData {
   staffEnterpriseCustomer: Types.EnterpriseCustomer;
 }
 
+export interface EnterpriseCourseEnrollment {
```

Comment:
nit: should this `interface` be extended further to reflect the `EnterpriseCourseEnrollment` serializer in the BFF ([source](https://github.com/openedx/enterprise-access/blob/106b52bb8632ef691ae3b884e280ab2a4175eef6/enterprise_access/apps/bffs/serializers.py#L255))?